unicodetools/data/security/dev/ReadMe.txt

```diff
@@ -1,7 +1,7 @@
 # Unicode Security Data (Confusables and Identifiers)
-# © 2023 Unicode®, Inc.
+# © COPY_YEAR Unicode®, Inc.
 # Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
 # For terms of use, see https://www.unicode.org/terms_of_use.html
 
-This directory contains the data files for Version 15.1.0 of
+This directory contains the data files for Version UNI_VER of
```

Comment:
Do we want `PUB_STATUS` for this file?

Reply:
In this PR I just want to templatize what we have.
Asmus is looking at revising the readmes overall, and Anne might want to chime in on copyright, terms of use, etc.
Let's make those kinds of content changes separately.